bql/semantic/expression.go

```diff
@@ -84,85 +84,41 @@ func (o OP) String() string {
 // evaluationNode represents the internal representation of one expression.
 type evaluationNode struct {
 	op OP
+
 	lB string
 	rB string
 }
 
-// comparisonForNodeLiteral represents the internal representation of a expression of comparison between a binding and a node literal.
-type comparisonForNodeLiteral struct {
-	op OP
-
-	lB  string
-	rNL string
-}
-
-func (e *comparisonForNodeLiteral) Evaluate(r table.Row) (bool, error) {
-	// Binary evaluation
-	eval := func() (*table.Cell, error) {
-		var (
-			eL *table.Cell
-			ok bool
-		)
-		eL, ok = r[e.lB]
-		if !ok {
-			return nil, fmt.Errorf("comparison operations require the binding value for %q for row %q to exist", e.lB, r)
-		}
-		return eL, nil
-	}
-
-	cs := func(c *table.Cell) string {
-		if c.L != nil {
-			return strings.TrimSpace(c.L.ToComparableString())
-		}
-		return strings.TrimSpace(c.String())
+// format cell to a trimmed comparable string
```

Comment:
I think you have to start the comment with the name of the function

Reply:
Done.